docs/adrs/010-data-model-definition.md

```diff
@@ -20,5 +20,6 @@ Accepted
 
 - Caikit users will be able to access model information through data models
 - Caikit users may face one extra level of indirection if they want to access mathematical representations of their data models results (e.g. arrays).
-- Caikit will provide the same data model per task [TODO: link to task ADR]. 
+- Caikit will provide the same data model per AI task [TODO: link to task ADR].
 - Data models will be backed by a common backend [TODO: link to backends, protobuf ADRs].
+- Caikit model authors will have to translate model inputs and outputs to work with data models. This has implications on usability and stability.
```

Comment:
I think s/Caikit model authors/Caikit module authors/

Reply:
I was trying to go for some consistency with the README personas provided here: https://github.com/caikit/caikit#user-profiles switched s/Caikit/AI/

Reply:
I think model makes more sense as we are looking at the profile from the high level go AI models.